config/routes.rb

```diff
@@ -1,8 +1,20 @@
+class SlackSlashSubcommandConstraint
```

Comment:
Constraint allows us to route different subcommands to different controllers instead of building up some big case statement based on subcommands.